src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GeometryEngine/SpatialRelationships/SpatialRelationships.xaml.cs

```diff
@@ -0,0 +1,176 @@
+        "GeometryEngine",
+        "This sample demonstrates how to use the GeometryEngine to evaluate the spatial relationships (for example, polygon a contains line b) of between geometries.",
+        "Tap a graphic to select it. The display will update to show the relationships with the other graphics.")]
+    public partial class SpatialRelationships
+    {
+        private GraphicsOverlay _graphicsOverlay;
+        private Graphic _polygonGraphic;
+        private Graphic _polylineGraphic;
+        private Graphic _pointGraphic;
+
+        public SpatialRelationships()
+        {
+            InitializeComponent();
+
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Configure the basemap
+            MyMapView.Map = new Map(Basemap.CreateTopographic());
+
+            // Create the graphics overlay and set the selection color
+            _graphicsOverlay = new GraphicsOverlay()
+            {
+                SelectionColor = System.Windows.Media.Colors.Yellow
+            };
+
+            // Add the overlay to the MapView
+            MyMapView.GraphicsOverlays.Add(_graphicsOverlay);
+
+            // Create the point collection that defines the polygon
+            PointCollection polygonPoints = new PointCollection(SpatialReferences.WebMercator)
+            {
+                new MapPoint(-5991501.677830, 5599295.131468),
+                new MapPoint(-6928550.398185, 2087936.739807),
+                new MapPoint(-3149463.800709, 1840803.011362),
+                new MapPoint(-1563689.043184, 3714900.452072),
+                new MapPoint(-3180355.516764, 5619889.608838)
+            };
+
+            // Create the polygon
+            Polygon polygonGeometry = new Polygon(polygonPoints);
+
+            // Define and apply the symbology
+            SimpleLineSymbol polygonOutlineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Windows.Media.Colors.Green, 2);
+            SimpleFillSymbol polygonFillSymbol = new SimpleFillSymbol(SimpleFillSymbolStyle.ForwardDiagonal, System.Windows.Media.Colors.Green, polygonOutlineSymbol);
+
+            // Create the graphic and add it to the graphics overlay
+            _polygonGraphic = new Graphic(polygonGeometry, polygonFillSymbol);
+            _graphicsOverlay.Graphics.Add(_polygonGraphic);
+
+            // Create the point collection that defines the polyline
+            var polylinePoints = new PointCollection(SpatialReferences.WebMercator)
+            {
+                new MapPoint(-4354240.726880, -609939.795721),
+                new MapPoint(-3427489.245210, 2139422.933233),
+                new MapPoint(-2109442.693501, 4301843.057130),
+                new MapPoint(-1810822.771630, 7205664.366363)
+            };
+
+            // Create the polyline
+            Polyline polylineGeometry = new Polyline(polylinePoints);
+
+            // Create the polyline and add it to the overlay
```

Comment:
Clarify some code comments (done for WPF - applies for the other SDK's as well):
```
// Create the polyline graphic and add it to the graphics overlay
```